packages/core/src/bin/ponder.ts

```diff
@@ -94,6 +95,7 @@ const startCommand = new Command("start")
     const cliOptions = {
       ...command.optsWithGlobals(),
       command: command.name(),
+      version: packageJson.version,
```

Comment:
I think this needs to get copied to a few more spots in this file. Or, does it get included automatically via `command.optsWithGlobals()` ?

Reply:
nice catch